### PR TITLE
Wait 10s to obtain a result from the searchByName API method

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

Wait 10s to obtain a result from the searchByName API method

## GUI diff

No difference.

- [x] **DONE**

## Documentation

- No documentation needed

- [x] **DONE**

## Test coverage
- Cucumber tests were refactored

- [x] **DONE**

## Links

Ports:
- Manager-4.3

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: testsuite/features/support/commonlib.rb [hunk 1]
```diff
@@ -254,7 +254,14 @@ def escape_regex(text)
 end
 
 def get_system_id(node)
-  $api_test.system.search_by_name(node.full_hostname).first['id']
+  result = []
```
Comment (Bischoff) [inline testsuite/features/support/commonlib.rb:257]: Add this comment:
```
# TODO - Remove this retrying code when the product issue is fixed:
```

**Review by Bischoff — CHANGES_REQUESTED** [review]
small change requested

**Review by ktsamis — APPROVED** [review]

**Review by nodeg — APPROVED** [review]
LGTM. The only valid remark is form Eric.

**Review by NamelessOne91 — APPROVED** [review]

**Review by Bischoff — APPROVED** [review]
thanks for the change

**Review by vandabarata — APPROVED** [review]

**Pull request merged**